tests/countries/test_sweden.py

```diff
@@ -173,6 +175,79 @@ def test_christmas(self):
     def test_sundays(self):
         self.assertSundays(Sweden)  # Sundays are considered holidays in Sweden.
 
+    def test_de_facto_holidays(self):
```

Comment:
There is no need for separate methods for categories; full coverage in the methods for individual holidays is sufficient.